.travis.yml

```diff
@@ -14,3 +14,11 @@ script:
 branches:
     only:
       - master
+
+deploy:
+    provider: npm
```

Comment:
I'm somewhat unclear, but you may need the email as in https://github.com/edx/edx-custom-a11y-rules/blob/master/.travis.yml#L17